compiler/rustc_session/src/config.rs

```diff
@@ -452,8 +452,10 @@ pub enum ExternLocation {
     /// The `CrateLoader` is responsible for loading these and figuring out
     /// which one to use.
     ///
+    /// These paths must be in canoncalized form.
```

Comment:
```suggestion
    /// These paths must be in canonicalized form.
```

Reply:
I don't think I spelled this word correctly anywhere I used it. This is actually also incorrect. It doesn't *have* to be canonicalized. I think I'll just remove this.